libs/databrickscfg/ops.go

```diff
@@ -129,6 +130,29 @@ func SaveToProfile(ctx context.Context, cfg *config.Config) error {
 	return configFile.SaveTo(configFile.Path())
 }
 
+func ValidateConfigAndProfileHost(cfg *databricks.Config, profile string) error {
+	configFile, err := config.LoadFile(cfg.ConfigFile)
+	if err != nil {
+		return fmt.Errorf("cannot parse config file: %w", err)
+	}
+	// Normalized version of the configured host.
+	host := normalizeHost(cfg.Host)
+	match, err := findMatchingProfile(configFile, func(s *ini.Section) bool {
+		return profile == s.Name()
+	})
+
+	if err != nil {
+		return err
+	}
+
+	hostFromProfile := normalizeHost(match.Key("host").Value())
```

Comment:
Can `.Key()` return nil?

Reply:
@pietern No, instead it returns a key with an empty value

Reply:
Ack, thanks.